sdk/appconfiguration/app-configuration/CHANGELOG.md

```diff
@@ -1,17 +1,15 @@
 # Release History
 
-## 1.5.0-beta.3 (Unreleased)
+## 1.5.0 (2023-11-07)
 
 ### Features Added
 
-### Breaking Changes
+- With the new API version `2023-10-01`, the configuration snapshot feature is generally available. Snapshots can be created with filters that determine which key-value pairs are contained within the snapshot, creating an immutable, composed view of the configuration store. The [`AppConfigurationClient`](https://docs.microsoft.com/javascript/api/@azure/app-configuration/appconfigurationclient) has some terminology changes from App Configuration in the portal. Snapshots are represented as `ConfigurationSnapshot` objects. See [`snapshot.ts`](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/appconfiguration/app-configuration/samples/v1-beta/typescript/src/snapshot.ts) for more information now how to use snapshots.
```

Comment:
Move this _"The [`AppConfigurationClient`](https://docs.microsoft.com/javascript/api/@azure/app-configuration/appconfigurationclient) has some terminology changes from App Configuration in the portal"_ to the key concepts in readme appropriately.